go.mod

```diff
@@ -313,3 +313,5 @@ replace (
 	github.com/go-ldap/ldap/v3 => github.com/YangKeao/ldap/v3 v3.4.5-0.20230421065457-369a3bab1117
 	github.com/pingcap/tidb/pkg/parser => ./pkg/parser
 )
+
+replace github.com/pingcap/kvproto => github.com/xzhangxian1008/kvproto v0.0.0-20231110025204-ddcf2223e2aa
```

Comment:
Is this expected?

Reply:
> Is this expected?

yes, I will change it to the latest tipb commit before pr is merged.